### ISSUE TITLE
CLI error after merging dev branch to master branch while live sync was on

### ISSUE THREAD

**Issue by MisterMunchkin** [NativeScript/NativeScript#5346]
Good Day, So I have this problem, while I was merging my development branch to my master branch I forgot to cancel the live sync thing in nativescript cli, and after the branch was merged, I start getting this error in my login page: 
JS: Missing Image with resourceID: res://logo_transparent_background
JS: Missing Image with resourceID: res://logo_transparent_background

I checked the res folder and the logo was still there, when when I tried to login in to the tab view I get this error:

System.err: com.tns.NativeScriptException:
System.err: Calling js method onCreateView failed
System.err:
System.err: ReferenceError: pl is not defined
System.err: File: "file:///data/data/org.nativescript.Unwind/files/app/tns_modules/nativescript-gif/gif.js, line: 17, column: 30
System.err:
System.err: StackTrace:
System.err:     Frame: function:'Gif.createNativeView', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/nativescript-gif/gif.js', line: 17, column: 31
System.err:     Frame: function:'ViewBase._setupUI', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 412, column: 35
System.err:     Frame: function:'', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 456,
column: 19
System.err:     Frame: function:'LayoutBaseCommon.eachChildView', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/layouts/layout-base-common.js', line: 125, column: 26
System.err:     Frame: function:'ViewCommon.eachChild', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/core/view/view-common.js', line: 699, column: 14
System.err:     Frame: function:'ViewBase._setupUI', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 455, column: 14
System.err:     Frame: function:'', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 456,
column: 19
System.err:     Frame: function:'LayoutBaseCommon.eachChildView', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/layouts/layout-base-common.js', line: 125, column: 26
System.err:     Frame: function:'ViewCommon.eachChild', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/core/view/view-common.js', line: 699, column: 14
System.err:     Frame: function:'ViewBase._setupUI', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 455, column: 14
System.err:     Frame: function:'', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 456,
column: 19
System.err:     Frame: function:'TabViewItemBase.eachChild', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/tab-view/tab-view-common.js', line: 75, column: 13
System.err:     Frame: function:'ViewBase._setupUI', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 455, column: 14
System.err:     Frame: function:'', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 456,
column: 19
System.err:     Frame: function:'', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/tab-view/tab-view-common.js', line: 167,
column: 17
System.err:     Frame: function:'TabViewBase.eachChild', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/tab-view/tab-view-common.js', line: 166, column: 19
System.err:     Frame: function:'ViewBase._setupUI', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 455, column: 14
System.err:     Frame: function:'', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 456,
column: 19
System.err:     Frame: function:'ContentView.eachChildView', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/content-view/content-view.js', line: 70, column: 13
System.err:     Frame: function:'PageBase.eachChildView', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/page/page-common.js', line: 206, column: 40
System.err:     Frame: function:'ViewCommon.eachChild', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/core/view/view-common.js', line: 699, column: 14
System.err:     Frame: function:'ViewBase._setupUI', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 455, column: 14
System.err:     Frame: function:'ViewBase._addViewCore', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 357, column: 18
System.err:     Frame: function:'ViewBase._addView', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 346, column: 14
System.err:     Frame: function:'FragmentCallbacksImplementation.onCreateView', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/frame/frame.js', line: 504, column: 24
System.err:     Frame: function:'FragmentClass.onCreateView', file:'file:///data/data/org.nativescript.Unwind/files/app/tns_modules/tns-core-modules/ui/frame/fragment.js', line: 27, column: 38
System.err:
System.err:     at com.tns.Runtime.callJSMethodNative(Native Method)
System.err:     at com.tns.Runtime.dispatchCallJSMethodNative(Runtime.java:1088)
System.err:     at com.tns.Runtime.callJSMethodImpl(Runtime.java:970)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:957)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:941)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:933)
System.err:     at com.tns.FragmentClass.onCreateView(FragmentClass.java:40)
System.err:     at android.app.Fragment.performCreateView(Fragment.java:2352)
System.err:     at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:995)
System.err:     at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1171)
System.err:     at android.app.BackStackRecord.run(BackStackRecord.java:815)
System.err:     at android.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1578)
System.err:     at android.app.FragmentManagerImpl$1.run(FragmentManager.java:483)
System.err:     at android.os.Handler.handleCallback(Handler.java:836)
System.err:     at android.os.Handler.dispatchMessage(Handler.java:103)
System.err:     at android.os.Looper.loop(Looper.java:208)
System.err:     at android.app.ActivityThread.main(ActivityThread.java:6267)
System.err:     at java.lang.reflect.Method.invoke(Native Method)
System.err:     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1063)
System.err:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:924)

please help me my deadline is next week and I'm not entirely sure what to do at this point, I tried to revert the commit but it gives me the same error.

### Which platform(s) does your issue occur on?
Occured in Android

### Please provide the following version numbers that your issue occurs with:
- CLI: 3.4.1
- Cross-platform modules: 3.4.0
- Runtime(s): tns-android: 3.4.1
                      tns-ios: 3.4.1
- Plugin(s):  "dependencies": {
    "nativescript-accordion": "^5.0.3",
    "nativescript-awesome-loaders": "^1.2.1",
    "nativescript-carousel": "^3.1.1",
    "nativescript-dom": "^2.0.0",
    "nativescript-drop-down": "^3.2.0",
    "nativescript-floatingactionbutton": "^4.1.2",
    "nativescript-gif": "^3.1.1",
    "nativescript-https": "^1.0.1",
    "nativescript-loading-indicator": "^2.4.0",
    "nativescript-loading-indicator-new": "^2.1.0",
    "nativescript-master-technology": "^1.1.1",
    "nativescript-modal-datetimepicker": "^1.0.3",
    "nativescript-paypal": "^2.6.9",
    "nativescript-pro-ui": "^3.2.0",
    "nativescript-reviews": "^1.0.4",
    "nativescript-snackbar": "^2.0.0",
    "nativescript-sqlite": "^1.1.11",
    "nativescript-theme-core": "1.0.4",
    "tns-core-modules": "^3.4.0"
  },
  "devDependencies": {
    "babel-traverse": "6.4.5",
    "babel-types": "6.4.5",
    "babylon": "6.4.5",
    "eslint": "4.6.1",
    "lazy": "1.0.11",
    "nativescript-dev-sass": "1.3.2",
    "node-sass": "4.5.3"
  }

### Please tell us how to recreate the issue in as much detail as possible.
The best approach would be to get your code running in the [NativeScript Playground](https://play.nativescript.org) and share the link with us, along with any additional details or steps to reproduce needed for examining the issue there. 
basically I commited to my development branch not know I still had the live sync on, I merged my dev to my master and github gave me an error: unable to unlink my debug.apk, after that I tried to run my project and it started giving me the error above in cli, I tried to revert the commit but it still gives me the same error

**Comment by MisterMunchkin** [comment]
Okay so I don't know why but I ran the project on an abandoned branch that wasn't messed up then ran the dev branch again and it started working fine again. oh my god for a minute there I thought I was gonna have nothing to present for my thesis
